packages/cfd/src/Helpers/compare-accounts-config.ts

```diff
@@ -16,17 +16,18 @@ const getHighlightedIconLabel = (
     const market_type_shortcode = market_type.concat('_', trading_platforms.shortcode);
     // Forex for these: MT5 Financial Vanuatu, MT5 Financial Labuan
     const forex_label =
-        ['financial_labuan', 'financial_vanuatu'].includes(market_type_shortcode) ||
+        market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_LABUAN ||
+        market_type_shortcode === MARKET_TYPE_SHORTCODE.FINANCIAL_VANUATU ||
```

Comment:
Using this comparision because there are types errors if we use `includes`

Reply:
can we offload `forex_label` to get it from API as well? This `if-else` looks rather patchy on the FE side